packages/babel-parser/test/fixtures/v8intrinsic/expression/await-expression/input.js

```diff
@@ -0,0 +1 @@
+async () => { await %StringParseInt("42", 10) }
```

Comment:
Can you also add a test for `yield`?

```js
function* fn() {
  yield %StringParseInt("42", 10);
}
```

We should ensure that it isn't parsed as
```js
function* fn() {
  (yield) % (StringParseInt("42", 10));
}
```

Reply:
Test case is added and passed.